common-pipeline/datarecord-metadata/datarecord-metadata-v3-schema.json

```diff
@@ -166,9 +169,40 @@
         "runtime_type": {
           "description": "Type of runtime",
           "type": "string"
+        },
+        "fraction_digits": {
+          "description": "Microsecond length for time and timestamp",
+          "type": "number"
+        },
+        "is_key": {
+          "description": "A field is key",
+          "type": "boolean"
+        },
+        "is_signed": {
+          "description": "Signed or Unsigned for number (bigint, integer, smallint, tinyint)",
+          "type": "boolean"
+        },
+        "item_index": {
+          "description": "Item record level",
+          "type": "number"
+        },
+        "min_length": {
+          "description": "Minimum length for fields (binary, bit, char, nchar)",
+          "type": "number"
+        },
+        "source_column_id": {
```

Comment:
I think this is to track where the column/field came from.  It might be from a merge node for example.  How do you see this being defined?  Is this enough to track the upstream field?  How about in the case of merge where you might have 2 nodes each having the same field?

Reply:
source_column_id could be link name or link GUID and column name combination. link has input / output where schema is saved.

Reply:
Should be renamed to `source_field_id`

Reply:
Matt, I have delivered following changes
1, renamed source_column_id to source_field_id
2, enhanced description of decimal_precision and decimal_scale to cover time and timestamp
3, removed fraction_digits